frontend/src/App.jsx

```diff
@@ -19,44 +18,33 @@ function App() {
                     <p className="usa-alert__text">This is a developer prototype, please do not judge me</p>
                 </div>
             </div>
-            <div className="flex">
-                <div className="one">
-                    <div className="rounded-box flex" id="title-box">
-                        <p>
-                            This is the OPRE
-                            <br />
-                            OPS system prototype.
-                        </p>
-                        <img alt="family" src={family} className="illustration" />
-                    </div>
-                </div>
-                <div className="two">
-                    <div className="rounded-box">
-                        <table>
-                            <caption>Workflows</caption>
-                            <tbody>
-                                <tr>
-                                    <th>Name</th>
-                                    <th>Path</th>
-                                </tr>
-                                <tr>
-                                    <td>Portfolios</td>
-                                    <td>
-                                        <Link to="portfolios">/portfolios</Link>
-                                    </td>
-                                </tr>
-                                <tr>
-                                    <td>CANs</td>
-                                    <td>
-                                        <Link to="cans">/cans</Link>
-                                    </td>
-                                </tr>
-                            </tbody>
-                        </table>
-                    </div>
-                </div>
-            </div>
-        </main>
+
+            <h1>This is the OPRE OPS system prototype.</h1>
```

Comment:
I think the word system is redundant as that's what the last S in the acronym stands for. 

Frankly, OPRE is as well as @kimschulke has mentioned but I think it's also clarifying to some extent---that one feels more more grey.

Reply:
Hmmm.  I'll change it in the next PR.